sections/05-adc-data-publishing.qmd

```diff
@@ -196,18 +236,32 @@ If you know that you will need to use this process for more than one dataset, we
 
-![](../images/submissions-form-data-file.png)
+Take a look at this recently made [data portal](https://arcticdata.io/catalog/portals/AIS). Originally, this user had only submitted the last listed dataset. That dataset contains over 700 rasters and shapefiles depicting vessel paths in the North Pacific at many different resolutions. Upon review, the submitter determined that it would be best if other users could download the data at specific resolution scales in addition to the whole collection of data. By creating separate datasets at the resolution scale, this user was able to organize their data in a structured way that makes it easy for future researchers to use their data. Furthermore, the user was able to house their related datasets under one roof using our portal system.
 
-Once there, you will see the following screen. In the Overview section, we recommend ***not*** editing the file name, and instead add a descriptive overview of the file. Once done, click the Attributes tab.
+::: callout-note
+To find out more about portals, visit <https://arcticdata.io/data-portals/>. You can view our catalog of portals [here](https://arcticdata.io/catalog/portals). We recommend reviewing the Distributed Biological Observatory's [portal](https://arcticdata.io/catalog/portals/DBO) to see what a more fleshed out portal can look like.
+:::
 
-![](../images/editor-18-socialsci.png)
+![](../images/kapsar-portal.png)
+
+This example also illustrates that users can create multiple datasets for a single NSF award. It's understandable that you may think you have to submit all your data for a given award under one dataset. However, we encourage you to be flexible in how you approach archiving and organizing your data with us as this user has been.
+
+When uploading your data files using SFTP, either through the command line or a GUI, we recommend creating folders with the same name as your dataset's title so that it is easier to associate the data with their associated metadata submission.
+
+::: callout-tip
+For more complex folder structures, it is wise to include a README file that explicitly walks users through the structure and provides a basic understanding of what is available. Generally speaking, we recommend structuring your data in an easy to understand way such that a README isn't completely necessary.
+:::
+
+### Self-Documenting Files
+
+NetCDF, .hdf, and .mat file types are all examples of "self-documenting" files. In the case of a NetCDF file, users can input the attribute name, attribute description, missing value codes, units, and more into the file itself. When these data are well-documented within themselves, it can save the user's time when they go to submit their data with us.
```

Comment:
remove this section, and add a summary instead. I'm moving this content up higher in the lesson and rewriting slightly

Reply:
added summary section and removed above info